src/webview_provider/queryResultPanel.ts

```diff
@@ -679,6 +679,17 @@ export class QueryResultPanel extends AltimateWebviewProvider {
       );
       return;
     }
+    const currentSize = getStringSizeInMb(
+      JSON.stringify(this._queryHistory),
+    );
+    // if current history size > 5MB, remove the oldest entry
+    if (currentSize > 5) {
+      this._queryHistory.pop();
```

Comment:
shift removes the first element in array. instead i want to remove the last element of the array, because i do insert the latest entry as first element